iothub/device/src/IotHubClientOptions.cs

```diff
@@ -79,6 +79,14 @@ public IotHubClientOptions(IotHubClientTransportSettings transportSettings)
         /// </remarks>
         public SdkAssignsMessageId SdkAssignsMessageId { get; set; } = SdkAssignsMessageId.Never;
 
+        /// <summary>
+        /// Sets the retry policy used in the operation retries.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to a near infinite exponential backoff. If set to null, will use <see cref="NoRetry"/> to perform no retries
```

Comment:
near infinite?

Please document other available built-in retry polies available to the user.

End the sentence with punctuation.